pkg/local_object_storage/shard/delete.go

```diff
@@ -40,7 +40,7 @@ func (s *Shard) Delete(prm DeletePrm) (DeleteRes, error) {
 		if s.hasWriteCache() {
 			err := s.writeCache.Delete(prm.addr[i])
 			if err != nil && !IsErrNotFound(err) {
-				s.log.Error("can't delete object from write cache", zap.String("error", err.Error()))
+				s.log.Info("can't delete object from write cache", zap.String("error", err.Error()))
```

Comment:
isn't it `Error`? `RO` mode is ok to be in `Info` but what about others?

Reply:
I don't think this should be an `error` level since garbage removal problems are not critical and don't require immediate attention. So, IMO we should leave no higher than `warn` level here. Thoughts @fyrchik?

Reply:
Both `Error` and `Warn` look ok to me.